### DISCUSSION TITLE
Question about ongoing project

### DISCUSSION THREAD

**Discussion by smallersign** [twbs/bootstrap#32743] [General]
As I can see BS Protect v5.0.0-beta2 is an ongoing process and almost **128** tasks are done. Is there a way to use partial source fixes files or it will accessible when rollout? 

I quite new here but not experienced yet.

**Comment by marispro** [comment]
Agree here, wish that beta releases come out more often. 
Last update "on 13 Oct 2020" might make feel somebody that package is not maintained.

**Comment by mdo** [comment]
When using Bootstrap via npm, you can use a Git tag or release as usual, or you can use a commit SHA. You're welcome to try that if it works for you. You can also download the source code from the main branch right on GitHub and put it to use for yourself.

We're working on getting a v4.6.0 out and then beta 2.

**Reply by smallersign** [reply]
Would you please share what will be commit npm command?
